contributing_to_docs/doc_guidelines.adoc

```diff
@@ -533,6 +533,11 @@ link:https://github.com/openshift/openshift-docs/pull/13878/files#diff-615ba1bf3
 example] of how this is applied.
 
 
+== Verification of your content
+All documentation changes must be verified by a QE team associate before merging. There are exceptions for typo level changes and other negotiated documentation sets and distributions.
+
+If a documentation change is due to a BugZilla bug, the bug should be put on ON_QA when you have a PR ready. Once QE ack is given (either in the bug or in the PR), the QE associate will move the bug status to VERIFIED, at which point, the associated PR may be merged.
```

Comment:
Probably less for this PR and maybe more for improving processes, but I'd say that it is a 50/50 chance that QE moves the bug status to VERIFIED. Often I get just the ack and have to move the bug from ON_QA to RELEASE_PENDING when I merge.

Reply:
Agreed. I often move the status myself. 
Should this say, "...the QE associate or writer moves the bug status to VERIFIED" at least until the process improves and QE always moves the status?

Reply:
- s/BugZilla/Bugzilla
- s/may be merged/can be merged

Reply:
I'm not sure if this is something we should recommend in our guidelines, but I think it's good practice. QE may only leave the ack on the issue, bug, or PR. In those cases, I'd suggest the writer leave a comment on whatever issue, bug, or PR the QE associate did not leave the ack so it's clear to reviewers that the ack is present. For example, if QE leaves the ack on the PR but not the Jira issue, the writer should leave a comment in the Jira issue along the lines of, "QE ack'd the PR".

Reply:
I agree it's good practice and it sounds like this works well for core OCP work, but I don't know if all CCS writing teams have the same level of QE support and are able to get acks on all changes? Do we need to consider that type of contribution here? I like what Brandi is saying maybe that could work for external team contributions?